ProductAnalysis.py

```diff
@@ -21,24 +21,74 @@
 #######################################################################
 
 
-def createVocab(reviewDataList, stopWords):
-  # Fill in method here
-  pass
+def createVocab(reviewDataList, productList, stopWords):
+  print(reviewDataList, productList, stopWords)
+  # Iterate through all the json files data and create vocabulary dictionary having the words and their associated counts
+  # Use parseWords to generate the tokenized terms
+  # Use nltk.FreqDist to generate term frequqnecies
+  allTerms, reviewList, reviewFreqDictList, productIdList, reviewIdList, reviewContentList = [], [], [], [], [], []
+  for r in range(len(reviewDataList)):
+    for review in reviewDataList[r]:#['Reviews']:
+      parsedWords = parseWords(review['fullText'], stopWords)
+      reviewFrequency = dict(nltk.FreqDist(parsedWords))
+      reviewFreqDictList.append(reviewFrequency)
+      reviewList.append(parsedWords)
+      reviewIdList.append(review['reviewId'])
+      productIdList.append(productList[r])
+      reviewContentList.append(review['fullText'])
+      allTerms += parsedWords
```

Comment:
should adding the len(parsedWords) or should it be adding it to a set of all the word terms and allTerms should be implemented as a set instead?

Reply:
I kept it consistent with HotelAnalysis. @ssubramanian90 can yo help answer Jackson's questions?

Reply:
Lets' keep it consistent for now to see if everything is working and then improve the code.